code/game/objects/items/weapons/material/kitchen.dm

```diff
@@ -192,3 +192,45 @@
 		user.Paralyse(2)
 		return
 	return ..()
+
+/obj/item/material/kitchen/utensil/knife/pocketbutterknife
+	name = "pocket butter knife"
+	desc = "An SCC Security approved modification of the H30V4 pocket knife."
+	desc_extended = "The H30V4B was developed to answer the immediate issue posed by the SCC not allowing pocket knives aboard the SCCV Horizon. Featuring a circular, \
+	rounded tip and notching in place of serration, it gets the job done. The knife is an employee exclusive, issued to Hephaestus crew either by purchase, or at \
+	request for 5+ year veteran employees."
+	icon = 'icons/obj/weapons.dmi'
+	icon_state = "pocketbutter"
+	attack_verb = list("poked", "jabbed", "schmeared")
+	var/on = FALSE
```

Comment:
DMDoc + this var needs a better name as you cannot turn "on" a knife (unless it's an electric knife, I guess?)